GoogleDataTransport/GDTCORLibrary/GDTCORConsoleLogger.m

```diff
@@ -16,40 +16,46 @@
 
 #import "GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCORConsoleLogger.h"
 
-volatile NSInteger GDTCORConsoleLoggerLoggingLevel = GDTCORLoggingLevelErrors;
+#import <GoogleUtilities/GULLogger.h>
+
+volatile NSInteger GDTCORConsoleLoggerLoggingLevel = GDTCORLoggingLevelWarnings;
```

Comment:
Looks like this is no longer used. Delete here and in header and update README reference.